src/main/java/org/sonarlint/intellij/config/project/SonarLintProjectAnalyzersPanel.java

```diff
@@ -0,0 +1,141 @@
+import java.util.Collections;
+import java.util.List;
+import javax.swing.JPanel;
+import javax.swing.JTable;
+import javax.swing.table.AbstractTableModel;
+import javax.swing.table.DefaultTableCellRenderer;
+import javax.swing.table.TableCellRenderer;
+import org.apache.commons.lang.StringUtils;
+import org.sonarlint.intellij.core.ProjectBindingManager;
+import org.sonarlint.intellij.util.SonarLintUtils;
+import org.sonarsource.sonarlint.core.client.api.connected.LoadedAnalyzer;
+
+public class SonarLintProjectAnalyzersPanel {
+  private static final Logger LOGGER = Logger.getInstance(SonarLintProjectAnalyzersPanel.class);
+  private final Project project;
+  private JBTable analyzerTable;
+  private JPanel panel;
+  private JPanel tablePanel;
+  private Model tableModel;
+
+  public SonarLintProjectAnalyzersPanel(Project project) {
+    this.project = project;
+    reload();
+  }
+
+  public JPanel getPanel() {
+    return panel;
+  }
+
+  public void reload() {
+    ProjectBindingManager bindingManager = SonarLintUtils.get(project, ProjectBindingManager.class);
+    try {
+      Collection<LoadedAnalyzer> loadedAnalyzers = bindingManager.getFacade().getLoadedAnalyzers();
+      tableModel.set(loadedAnalyzers);
+    } catch(Exception e) {
+      LOGGER.error(e);
+      tableModel.set(Collections.emptyList());
+    }
+  }
+
+  private void createUIComponents() {
+    tableModel = new Model();
+    analyzerTable = new JBTable(tableModel);
+    analyzerTable.setShowGrid(false);
+    analyzerTable.setIntercellSpacing(JBUI.emptySize());
+    analyzerTable.getEmptyText().setText("No information");
+    analyzerTable.setDragEnabled(false);
+    analyzerTable.setShowVerticalLines(false);
+    analyzerTable.getTableHeader().setReorderingAllowed(false);
+    analyzerTable.enableInputMethods(false);
+    analyzerTable.setDefaultRenderer(String.class, new NoFocusCellRenderer(new DefaultTableCellRenderer()));
+
+    final ToolbarDecorator decorator = ToolbarDecorator.createDecorator(analyzerTable);
+    tablePanel = decorator.createPanel();
+  }
+
+  private class NoFocusCellRenderer extends DefaultTableCellRenderer {
+    private final TableCellRenderer delegate;
+
+    public NoFocusCellRenderer(TableCellRenderer delegate) {
+      this.delegate = delegate;
+    }
+
+    public Component getTableCellRendererComponent(JTable table, Object value, boolean isSelected, boolean hasFocus, int row, int column) {
+      return delegate.getTableCellRendererComponent(table, value, isSelected, false, row, column);
+    }
+  }
+
+  private static class Model extends AbstractTableModel {
+    private List<LoadedAnalyzer> rows = new ArrayList<>();
+
+    @Override public int getRowCount() {
+      return rows.size();
+    }
+
+    @Override public int getColumnCount() {
+      return 2;
+    }
+
+    @Override
+    public boolean isCellEditable(int row, int column) {
+      return false;
+    }
+
+    public Class<?> getColumnClass(int columnIndex) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add the "@Override" annotation above this method signature [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1161)